indico/htdocs/sass/partials/_itempicker.scss

```diff
@@ -0,0 +1,76 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+.item-picker-qtip {
+    @include border-radius();
+    @include single-box-shadow();
+
+    border: thin solid lighten($gray, $color-variation);
+    box-shadow: 0 6px 12px lighten($gray, $color-variation);
```

Comment:
Maybe it's not a bad moment for adding a new `box-shadow` mixin. `single-box-shadow` was meant for i-box like elements that standout over the background but just slightly. For clearly hovering elements like the itempicker we could a have another mixin that reflects this better.